AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -142,7 +143,8 @@ define([
                         displayedRecord['is_licensed'] = 1;
                         displayedRecord['is_licensed_locally'] = 1;
                     }
-
+                    this.source().set('params.t ', Date.now());
+                    this.preview().hide();
```

Comment:
The preview should not be hidden after saving the image

Reply:
If we don't close the preview then we'll get the `record.series is not a function` error. So what should be the best solution to overcome this issue?